apps/main/lib/main/views/home.rb

```diff
@@ -0,0 +1,18 @@
+require "main/import"
+require "main/view/controller"
+
+module Main
+  module Views
+    class Home < Main::View::Controller
+      include Main::Import["repositories.articles_repo"]
+
+      configure do |config|
+        config.template = "home"
+      end
+
+      expose :articles do
+        articles_repo.home_page_listing
```

Comment:
> does "repositories.articles_repo" do a require_relative "../../repositories/articles_repo.rb"?

No, it does `require "main/repositories/articles_repo"` because we have load paths set

> variable articles_repo name is extracted from the string? and automagically be assigned with Main::Repositories::ArticlesRepo class loaded from the .rb ?

Yes, dependency identifier is used to determine namespaces and class names